content/en/contribute/medic/product-development-process/ux-research-repo/baserow-feedback.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Baserow Feedback"
```

Comment:
I think we should just delete this entire page as we aren't going to keep track of it/update it here.

Reply:
Do you mean that this page should stay in Notion, or should we remove it completely?

Reply:
We can either leave it in Notion or move it to a google sheet (or baserow itself) if we want to totally get rid of the notion page.  I'm actually hoping we'll close off most of these during a hackathon-esque session in Zanzi.

Reply:
@craig-landry has reviewed the Baserow documentation via [this PR](https://github.com/medic/cht-docs/pull/941), and removed this page completely. Thanks Craig!